lxd/device/device_utils_network.go

```diff
@@ -560,6 +560,40 @@ func networkSetupHostVethLimits(m deviceConfig.Device) error {
 		}
 	}
 
+	networkPriority := 0
+	if m["limits.priority"] != "" {
+		networkPriority, err = strconv.Atoi(m["limits.priority"])
+		if err != nil {
+			return fmt.Errorf("Failed to parse limits.priority (%s): %s", m["limits.priority"], err)
```

Comment:
We tend to use `%q` and drop the `()` for input strings that have an error, that way Go will wrap them in double quotes so differentiating from the error message and the actual input value is clear and consistent.

Reply:
Thanks!

Have fixed.